main/options/options.cc

```diff
@@ -666,6 +668,8 @@ void readOptions(Options &opts,
         opts.lspDocumentSymbolEnabled =
             enableAllLSPFeatures || raw["enable-experimental-lsp-document-symbol"].as<bool>();
         opts.lspSignatureHelpEnabled = enableAllLSPFeatures || raw["enable-experimental-lsp-signature-help"].as<bool>();
+        opts.lspCancelableSlowPathEnabled =
+            enableAllLSPFeatures || raw["enable-experimental-lsp-cancelable-slow-path"].as<bool>();
```

Comment:
Note that we enable all experimental features in sorbet.run, but I think it pretends to be single threaded:

https://github.com/sorbet/sorbet/blob/4bef9ab4572a5e4eef714cca9c7586de94eb051f/emscripten/main.cc#L23

idk quite how this interacts with your changes, but flagging just in case you wanted to take a look and confirm that this PR probably won't break sorbet.run

Reply:
I actually didn't make `enableAllExperimentalFeatures` flip this flag on since it wasn't relevant. Tests in test_corpus take the same path as sorbet.run. I can manually check sorbet.run before merging, though.

Reply:
can we make `enableAllExperimentalFeatures` enable it?
otherwise this might cause confusion

Reply:
Now that multithreaded mode isn't special, sure.